packages/manual/linux.d/patches/5.10.78/kernel/fpga-ofs.patch

```diff
@@ -0,0 +1,12501 @@
+diff --git a/Documentation/ABI/testing/sysfs-bus-dfl-devices-emif b/Documentation/ABI/testing/sysfs-bus-dfl-devices-emif
```

Comment:
just symlink to `../../5.10.70/kernel/fpga-ofs.patch`?

Reply:
Agreed. If we have the same patch for multiple versions, we should symlink to the first version of the patch file.

5.10.78 was work in progress - removed it.